website/docs/administration/smtp/index.md

```diff
@@ -20,10 +20,24 @@ You can configure the SMTP server settings in the **Mail Delivery** page.
 
 Other providers such as [SendGrid](https://sendgrid.com/), [Mailgun](https://www.mailgun.com/) or [Resend](https://resend.com) can be configured by providing the SMTP server details. You can find the SMTP server details in the respective provider's documentation.
 
+## Use Self-Signed Certificate
+
+Self-signed Certificate can also be used in Tabby Mail Delivery,
+to use self-signed certificate, you can add an environment variable `TABBY_WEBSERVER_EMAIL_CERT`
+before running Tabby, and set its value to the certificate, for example, in bash:
+
+```bash
+export TABBY_WEBSERVER_EMAIL_CERT=$(cat ./self-signed.crt)
+```
+
+Due to some limitations on macOS, the self-signed certificate is not working under macOS,
+you can add the certificate to the macOS `Keychain Access` and set it to `Always Trusted` if needed.
+Please refer to [Apple Support](https://support.apple.com/en-hk/guide/keychain-access/kyca11871/mac)
+for more detail.
```

Comment:
```suggestion
```

Given that the primary use case of SMTP is for IT/infrastructure administrators (in a Linux environment), I believe it's safe to exclude this part from the scope of the documentation.